### PR TITLE
T5-288 Update shopify form

### PR DESCRIPTION
**Pull request by naeluh**

### CODE REVIEW

File: src/templates/insights.js [hunk 1]
```diff
@@ -105,6 +105,31 @@ const Insights = ({ data }) => {
...
+              minHeight='none'
+              height='100px'
+              padding='0 20px 50px'
+            >
+              <ButtonFormDownload
+                filepath={
+                  post.relationships.field_e_book_file.localFile.publicURL
+                }
+                text='Access E-book'
+                header='Submit your email to access our free ebook'
```
Comment (jds-tag) [inline src/templates/insights.js:118]: Do we want a period here?

File: src/templates/insights.js [hunk 2]
```diff
@@ -105,6 +105,31 @@ const Insights = ({ data }) => {
...
+                confirmMessage='Thanks for you submission!'
```
Comment (jds-tag) [inline src/templates/insights.js:119]: "Thank you for your submission!"

**Review by jds-tag — CHANGES_REQUESTED** [review]
Grammar nits but lgtm otherwise!

File: src/templates/insights.js [hunk 3]
```diff
@@ -115,8 +115,8 @@ const Insights = ({ data }) => {
                   post.relationships.field_e_book_file.localFile.publicURL
                 }
                 text='Access E-book'
-                header='Submit your email to access our free ebook'
-                confirmMessage='Thanks for you submission!'
+                header='Submit your email to access our free ebook.'
+                confirmMessage='Thank you for you submission!'
```
Comment (jds-tag) [inline src/templates/insights.js:119]: `for your submission` (not "you").

**Review by jds-tag — APPROVED** [review]
LGTM!

File: src/components/SplitHeader/SplitHeader.js [hunk 4]
```diff
@@ -0,0 +1,434 @@
...
+                h3 {
+                  color: ${fontColor};
+                }
+                .thanks {
+                  margin: 0;
+                  justify-self: center;
+                  color: ${fontColor};
+                  padding: 0;
+                }
+
```
Comment (jds-tag) [inline src/components/SplitHeader/SplitHeader.js:318]: This new line is cool but there are no new lines anywhere else. Should this be consistent?

File: src/components/SplitHeader/SplitHeader.js [hunk 5]
```diff
@@ -0,0 +1,434 @@
...
+  useEffect(() => {
+    setLoaded('block');
+  }, []);
+
+  const innerSectionCSS = css`
+    ${container.max}
+    display: flex;
+    flex-direction: column-reverse;
+
+    div.left {
```
Comment (jds-tag) [inline src/components/SplitHeader/SplitHeader.js:216]: Not a dealbreaker but does `left` and `right` make the most sense semantically if they stack on mobile?

File: src/components/SplitHeader/SplitHeader.js [hunk 6]
```diff
@@ -0,0 +1,434 @@
...
+                <div
+                  dangerouslySetInnerHTML={{
+                    __html: summary,
+                  }}
+                />
+              )}
+            </FullWidthSection>
+            <ButtonFormDownload
+              filepath={ebook.localFile.publicURL}
+              text='Download E-book'
```
Comment (jds-tag) [inline src/components/SplitHeader/SplitHeader.js:342]: Is it `ebook` or `E-book`? Make it consistent.

**Review by jds-tag — COMMENTED** [review]
Some nits.

File: src/components/SplitHeader/SplitHeader.js (hunk 6 above)
Comment (naeluh) [reply src/components/SplitHeader/SplitHeader.js:342]: Updated the names thanks!

File: src/components/SplitHeader/SplitHeader.js (hunk 5 above)
Comment (naeluh) [reply src/components/SplitHeader/SplitHeader.js:216]: Updated the names thanks!

File: src/components/SplitHeader/SplitHeader.js (hunk 4 above)
Comment (naeluh) [reply src/components/SplitHeader/SplitHeader.js:318]: Update the spacing thanks!

**Review by jds-tag — APPROVED** [review]
LGTM! Just noting that `E-book` or `E-Book` is the appropriate usage (saw it in an Acquia email this morning) so just make sure anytime `E-book` is written out it has the hyphen. Thanks!

**Pull request merged**